src/components/atoms/ImageInput/ImageInput.tsx

```diff
@@ -97,11 +101,37 @@ export const ImageInput: React.FC<ImageInputProps> = ({
 
   const labelClasses = classNames("ImageInput__container", {
     "ImageInput__container--error": !!error?.message,
-    "ImageInput__container--small": small,
     "ImageInput__container--disabled": loading,
     "mod--hidden": isInputHidden || !imageUrl,
   });
 
+  console.log("imgUrl", fileUrl);
+
+  const wrapperClasses = classNames(
+    "ImageInput__wrapper justify-center items-center flex bg-cover",
+    {
+      "w-full text-white rounded-md": variant === "wide",
+      "w-48 h-48 text-white rounded-full flex-shrink-0 mx-auto":
+        variant === "round",
+    }
+  );
```

Comment:
[NOTE] I'm guessing we're still not done with this component, but one day soon we should either replace it or put all its classes in order, separate file etc.

Reply:
added tailwind module `src/components/admin/ImageInput/ImageInput.tailwind.ts`: https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0